src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```
@@ -117,6 +117,12 @@ export class ProsemirrorAdapter {
     @Prop()
     triggerCharacters: TriggerCharacter[] = [];
 
+    /**
+     * Set to `true` to allow images to be pasted into the editor.
+     */
+    @Prop()
+    allowImages: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)